Dockerfile

```diff
@@ -0,0 +1,6 @@
+# syntax=docker:dockerfile:1
```

Comment:
This doesn't actually do anything. Can you elaborate on why this is added?

Reply:
This is the first time I write dockerfile, so I just mimic what others have in a dockerfile. I've removed this since this isn't do anything.

Reply:
That's fine but these comments aren't used anywhere in https://carpentries-incubator.github.io/docker-introduction/advanced-containers/index.html. The goal as laid out in Issue #5 was to work through the Carpentries module and then build your own Dockerfile, rather than trying to find existing files to copy. If The Carpentries module didn't have enough examples you can look at this module starting with https://hsf-training.github.io/hsf-training-docker/05-dockerfiles/index.html or ask questions.